pkg/distributors/contracts/MultiRewards.sol

```diff
@@ -381,11 +386,29 @@ contract MultiRewards is IMultiRewards, IDistributor, ReentrancyGuard, Temporari
     function exit(IERC20[] calldata pools) external {
         for (uint256 p; p < pools.length; p++) {
             IERC20 pool = pools[p];
-            unstake(pool, _balances[pool][msg.sender]);
+            unstake(pool, _balances[pool][msg.sender], msg.sender);
         }
         getReward(pools);
     }
 
+    /**
+     * @notice Allows a user to unstake all their bpt to exit pools
+     *         and transfers them all the rewards
+     */
+    function exitWithCallback(
+        IERC20[] calldata pools,
+        address callbackContract,
+        bytes calldata callbackData
+    ) public returns (bytes memory) {
+        for (uint256 p; p < pools.length; p++) {
+            IERC20 pool = pools[p];
+            unstake(pool, _balances[pool][msg.sender], callbackContract);
```

Comment:
```suggestion
            unstake(pools[p], _balances[pool][msg.sender], callbackContract);
```

I would inline this as I don't think it really hampers readability at all. Feel free to ignore.

Reply:
So it would actually be  `unstake(pools[p], _balances[pools[p]][msg.sender], callbackContract);` if inlined, which I find a bit confusing